contracts/test/CompoundGovernor.t.sol

```diff
@@ -24,7 +24,7 @@ contract CompoundGovernorTest is Test, CompoundGovernorConstants {
     }
 
     function testInitialize() public view {
-        assertEq(governor.quorumVotes(), INITIAL_QUORUM);
+        assertEq(governor.quorum(block.timestamp), INITIAL_QUORUM);
```

Comment:
should be `block.number`?

Reply:
Done: 878d2b65c000cdd9364e066f4b693c2273422a47

Reply:
This was reverted